[2023-10-26T07:12:54.887+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load_data manual__2023-10-26T07:02:00.120485+00:00 [queued]>
[2023-10-26T07:12:54.902+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load_data manual__2023-10-26T07:02:00.120485+00:00 [queued]>
[2023-10-26T07:12:54.903+0000] {taskinstance.py:1359} INFO - Starting attempt 3 of 3
[2023-10-26T07:12:54.928+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): load_data> on 2023-10-26 07:02:00.120485+00:00
[2023-10-26T07:12:54.937+0000] {standard_task_runner.py:57} INFO - Started process 137 to run task
[2023-10-26T07:12:54.942+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load_data', 'manual__2023-10-26T07:02:00.120485+00:00', '--job-id', '474', '--raw', '--subdir', 'DAGS_FOLDER/tugaskel8_dag.py', '--cfg-path', '/tmp/tmpq3dqevkh']
[2023-10-26T07:12:54.947+0000] {standard_task_runner.py:85} INFO - Job 474: Subtask load_data
[2023-10-26T07:12:55.014+0000] {task_command.py:415} INFO - Running <TaskInstance: etl_dag.load_data manual__2023-10-26T07:02:00.120485+00:00 [running]> on host 52f0e9dbe5c3
[2023-10-26T07:12:55.348+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kel8' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-26T07:02:00.120485+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-26T07:02:00.120485+00:00'
[2023-10-26T07:12:55.431+0000] {base.py:73} INFO - Using connection ID 'db_polusiudara' for task execution.
[2023-10-26T07:12:57.935+0000] {python.py:194} INFO - Done. Returned value was: Sukses Memasukkan Data
[2023-10-26T07:12:57.968+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=load_data, execution_date=20231026T070200, start_date=20231026T071254, end_date=20231026T071257
[2023-10-26T07:12:58.012+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-26T07:12:58.032+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
